radix-engine-tests/benches/radiswap.rs

```diff
@@ -36,7 +36,7 @@ fn bench_radiswap(c: &mut Criterion) {
     let component_address: ComponentAddress = test_runner
         .execute_manifest(
             ManifestBuilder::new()
-                .lock_fee(account2, 50u32.into())
+                .lock_fee(account2, 500u32.into())
```

Comment:
Maybe move this value to a constant, so future changes will be easier.

Reply:
That's a good idea. If I have to change it one more time, will make it a constant :)